examples/ArduinoJson/ArduinoJson.ino

```diff
@@ -60,7 +60,8 @@ void compute_mediaList()
   // Print values.
   const char *id = media["id"];
   const char *folder = media["media"][0]["d"];
-  JsonObject object = media["media"][0]["fs"];
+  //JsonObject object = media["media"][0]["fs"];
```

Comment:
Can you test your code with JsonObject instead of JsonArray? I am not sure this is an array
See [this](https://stackoverflow.com/questions/12289844/difference-between-jsonobject-and-jsonarray), even if it is from java it should be the same

Reply:
ok I will try again, JsonObject was not working

Reply:
Hi Aster, I made some tests this mornig. Object not working, 1 photo + 1 Video on the gopro (2 medias), see images : 
![Capture array](https://user-images.githubusercontent.com/47244003/121998035-889ca100-cdab-11eb-8818-de989bb32477.PNG)
![Capture object](https://user-images.githubusercontent.com/47244003/121998039-89353780-cdab-11eb-926f-0022afad7882.PNG)

Reply:
okkkk you were right! Probably when I tested it in the first time I haven't run many tests with json